.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 script:
   # Build docker image
   - cd 2.3/
-  - docker build -t maestrano/web-ruby:travis . > /dev/null 2>&1
```

Comment:
can you plz explain why this isn't required any more?

Reply:
@cesar-tonnoir looks like this was removed so we can see the docker build logs, also the `2>&1` was masking the exit code 🤔- see https://github.com/maestrano/docker-web-ruby/commit/43f2e36d90369669c60d8be3468b4372006e0145

Reply:
Yes exactly. I removed it to be able to see the docker logs when debugging and decided to keep it.

- `> /dev/null` redirects `stdout` to `/dev/null/
- `2>&1` redirects `stderr` to where we're redirecting `stdout`